### ISSUE TITLE
[FR] command bar support

### ISSUE THREAD

**Issue by garbosz** [franciskafieh/obsidian-list-modified#117]
## First Steps

-   [x] I've ensured that this feature request **does not already exist** by [using the search feature](https://github.com/franciskafieh/obsidian-list-modified/issues?q=search+issues+here)
-   [x] I've read the documentation (see [the README](https://github.com/franciskafieh/obsidian-list-modified#readme))
-   [x] I'm on the latest release of obsidian-list-modified

## Describe the desired feature
i think it would be useful, especially when troubleshooting the plugin, to have a command available in the command bar to manually refresh the listed files, as well as commands to manually trigger note reset and to clear/rebuild the current queue of listed files. 
## Possible alternatives
mostly what i do now is have a test file that has a "force modify" check box property, so whenever i want the plugin to manually trigger(whether it be troubleshooting the bug, testing format changes, or i accidently deleted or pasted something wrong) i would just toggle it and it would update, and for clearing rebuilding the queue of listed files i just do the suggested trick of modifying the data inside the code from vscode. 
## Additional info?
basically expose some of the useful functions of the plugin to the command bar

**Comment by franciskafieh** [comment]
> manually refresh the listed files

another possible workaround for now is just toggling "auto create (one of the dividers)" back and forth. This will update the modified files list.


Instead of having this as a command bar option, I will probably put it under debug settings since I don't think most normal users should have a usecase for this unless there is a bug.
